cfgov/jinja2/v1/es/es-base.html

```diff
@@ -184,13 +174,15 @@ <h1><a class="brand-logo" href="/es/"><img src="{{ static('nemo/_/img/cfpb-logo-
 
 </footer><!-- .wrapper-footer -->
 
+{% endblock body %}
+
+
+{% block javascript %}
```

Comment:
It's `{% block javascript scoped %}` in the english template. Not totally sure on why we added the `scoped` (@sebworks ?), but do these need to match?

Reply:
yeah i wasn't sure if the `scoped` does anything either but the js loads as expected in both templates so i left it this way.